cleanlab/datalab/internal/issue_manager/null.py

```diff
@@ -140,3 +140,66 @@ def collect_info(self, null_tracker: np.ndarray) -> dict:
         issues_dict = {**average_null_score, **most_common_issue, **column_impact}
         info_dict: Dict[str, Any] = {**issues_dict}
         return info_dict
+
+    @classmethod
+    def report(cls, *args, **kwargs) -> str:
```

Comment:
### Why We Changed the Report Method
For the first time, the class method has been extended for an issue manager. This extension lets users know that `Datalab` can do more checks on features after null values are fixed.

### Heads Up About Null Values
If there are null values in features, it can cause issues. Other issue managers that need these features will only work right if null values are fixed first (because some models don't work with null values).

### What You Should Do
**First, fix the null value problems we found.** After that, run `Datalab` again on the data you amended to ensure it works fine.

### More Help on the Way
Guides on how to deal with null values in different situations will be made separately.